lib/smart_answer_flows/coronavirus-find-support/outcomes/_feeling_unsafe.erb

```diff
@@ -0,0 +1,61 @@
+## Feeling unsafe
+
+### If you do not feel safe where you live or you’re worried about someone else
+
+If you’re in immediate danger call 999 and ask for the Police.
+
+If you’re in danger and unable to talk on the phone, call 999, then press 55 when prompted.
+
+Call the National Domestic Abuse Helpline on <a href="tel:0808 2000 247" class="govuk-link">0808 2000 247</a>, you can get advice in different languages. You can also <a href="https://www.nationaldahelpline.org.uk/Chat-to-us-online" class="govuk-link">get help online</a>.
```

Comment:
These links need a class defined. So need to be HTML. Hmm.

Reply:
Thanks @reggieb - will turn into Markdown.